build.gradle

```diff
@@ -38,7 +38,7 @@ ext {
     buildToolsVersion = '28.0.3'
 
     // App dependencies
-    supportLibraryVersion = '1.1.0'
+    supportLibraryVersion = '1.0.0'
```

Comment:
I changed the supportLibraryVersion because version 1.1.0 disabled scrolling as discussed here -
https://gitter.im/openMF/mifos